python/src/DistFunc_doc.i.in

```diff
@@ -583,185 +675,1304 @@ Examples
+    Tail flag.
+    
+    If True, the complementary CDF is computed.
+
+Returns
+-------
+p : float
+    The CDF or the complementary CDF at x.
+
+Notes
+-----
+The Kolmogorov distribution is the distribution of :math:`D_n = \sup_y \|F(y) - F_n(y) \|`
+where :math:`F_n` is the empirical cumulative distribution built from :math:`n` independent and identically
+distributed observations and :math:`F` the cumulative distribution  to be tested.
+
+We use the algorithm and the selection  strategy is described in [simard2011]_.
+
+Examples
+--------
+>>> import openturns as ot
+>>> cdf = ot.DistFunc.pKolmogorov(100, 0.1)"
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::pSpearmanCorrelation
+"Cumulative distribution function  of a the Spearman correlation distribution.
+
+Parameters
+----------
+n : int
+    The number of distinct ranks.
+rho : float
+    Location.
+tail : bool, default=False
+    Tail flag. If True, the complementary CDF is computed.
+ties : bool, default=False
+    Indicates whether ties in the samples are taken into account.
+
+Returns
+-------
+p : float
+    The CDF or its complementary of the Spearman statistics distribution.
+
+Notes
+-----
+This method enables to test whether an observed value of the Spearman correlation :math:`\rho`
+is significantly different from zero. It implements the distribution of the statistics under the
+null hypothesis where the Spearmann correlation is assumed to be not significantly different from 0.
+
+The CDF value is the probability that the statistics is less or equal
+than the value calculated from :math:`\rho`. The complementary CDF value is the p-value.
+
+When there is no ties, we use the tabulated exact distribution of the statistics for small sample
+size thanks to the values given in [luke]_ and the Edgeworth
+expansion for the other sample sizes.
+
+When there is some ties, we use the Student asymptotic approximation.
+
+Examples
+--------
+>>> import openturns as ot
+>>> cdf = ot.DistFunc.pSpearmanCorrelation(100, 0.1)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::qDickeyFullerTrend
+"Quantile of the Dickey-Fuller statistics.
+
+Parameters
+----------
+p : float
+    The probability.
+tail : bool, default=False
+    Tail flag.
+
+    If True, the quantile associated to :math:`(1-p)` is computed.
+
+Returns
+-------
+q : float
+    The quantile of order :math:`p` or :math:`(1-p)`.
+
+Notes
+-----
+
+Refer to :ref:`dickey_fuller`.
+
+We denote :math:`(X_1, \hdots, X_n)` the data,  :math:`W(r)` the Wiener process and :math:`W^{a}(r) = W(r) - \int_{0}^{1} W(r)\di{r}`,
+:math:`W^{b}(r) = W^{a}(r) - 12 \left(r - \frac{1}{2} \right) \int_{0}^{1} \left(s - \frac{1}{2} \right) W(s)\di{s}`.
+
+The Dickey-Fuller statistics is the asymptotic distribution of the Student statistics used to test the model with drift and trend in a time series: 
```

Comment:
I feel like you are trying to say too many things in a single sentence and the meaning is unclear to me.

Reply:
yes!